pkg/mcp/snapshot/snapshot.go

```diff
@@ -86,6 +88,7 @@ type StatusInfo struct {
 	node                 *mcp.SinkNode
 	lastWatchRequestTime time.Time // informational
 	watches              map[int64]*responseWatch
+	synced               map[string]map[string]bool
```

Comment:
Can you write a comment explaining what the keys are? It looks like this is snapshot/peerAddr. However, this should already be a per-connection data structure. Am I missing anything?

Reply:
the synced structure is {"Collection": {"peerAddress": synced|nosynced}}. Here is an example {"istio/config/v1alpha2/legacy/metrics": {"10.10.0.1:23456": true|false}}

Reply:
Please add the comment to the PR. My point is, will there ever be one key for the inner map? This data structure is per-connection, right?

Reply:
no. it will have multiple key for the inner map. it can be  {"istio/config/v1alpha2/legacy/metrics": {{"10.10.0.1:23456": true}, {"10.10.10.100:12345": true}}}

Reply:
It looks like node in StatusInfo and its comment is wrong (which caused me to ask this question in the first place).

Can you update those as part of your changes? StatusInfo should not have node (it is not used). Also, the comment mentions that it is for a "remote node" whereas it should be for a group.

Reply:
lgtm otherwise.

Reply:
sure. updated my PR.